images/arch/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/archlinux/base:latest
```

Comment:
I think "archlinux" is more official. It's referenced from https://www.archlinux.org/download/

```suggestion
FROM docker.io/archlinux:latest
```